db/query.go

```diff
@@ -408,20 +407,8 @@ func (context *DatabaseContext) buildChannelsQuery(channelName string, startSeq
 	return channelQueryStatement, params
 }
 
-func (context *DatabaseContext) QueryResync() (sgbucket.QueryResultIterator, error) {
-
-	if context.Options.UseViews {
-		opts := Body{"stale": false, "reduce": false}
-		opts[QueryParamStartKey] = []interface{}{true}
-		return context.ViewQueryWithStats(DesignDocSyncHousekeeping(), ViewImport, opts)
-	}
-
-	// N1QL Query
-	var importQueryStatement string
-	importQueryStatement = replaceSyncTokensQuery(QueryResync.statement, context.UseXattrs())
-	importQueryStatement = replaceIndexTokensQuery(importQueryStatement, sgIndexes[IndexAllDocs], context.UseXattrs())
-
-	return context.N1QLQueryWithStats(QueryTypeResync, importQueryStatement, nil, gocb.RequestPlus, QueryResync.adhoc)
+func (context *DatabaseContext) QueryResync(limit int, startSeq, endSeq uint64) (sgbucket.QueryResultIterator, error) {
+	return context.QueryChannels("*", startSeq, endSeq, limit, false)
```

Comment:
I realize there are other places in query.go that aren't using it, but the existing UserStarChannel constant can be used here instead of "*" (and maybe fix the other spots while you're at it).

Reply:
Fixed in all the prod code cases I could spot. I left any existing tests as I thought the explicit "*" was fine there and if they do break in the future it'll be fairly obvious.